tests/rbd/rbd_utils.py

```diff
@@ -649,3 +673,24 @@ def initial_rbd_config(**kw):
         rbd_obj.update({"rbd_ecpool": rbd_ecpool})
 
     return rbd_obj
+
+
+def execute_dynamic(rbd, test, results: dict, **kwargs):
+    """
+    Executes the test specified in test parameter with inputs args and returns results
+    Args:
+        rbd: rbd object
+        test: test to be executed
+        results: test result
+        **kwargs: input args required for the test
+
+    Returns:
+        results updated to results dict
+    """
+    method = getattr(rbd, test)
+    rc = method(**kwargs)
+    log.info(f"Return value for execution of method: {test} is {rc}")
```

Comment:
Instead of return value message here, can we provide the method gets passed or failed as message in log inside  `if else` condition.

Reply:
Agree with you @VasishtaShastry , the return values are 0 and 1 here. I have not added any other return values afaik.

Reply:
@sunilangadi2 That log info on whether the test passed or failed is already being sent at the end of test_parallel module, so I think again adding an if else for that here would be redundant. Please let me know your thoughts.